src/world/area_omo/omo_12/DDF730.c

```diff
@@ -1,3 +1,21 @@
 #include "omo_12.h"
 
 INCLUDE_ASM(s32, "world/area_omo/omo_12/DDF730", func_80240420_DDF730);
+/*
+ApiStatus N(func_80240420_DDF730)(ScriptInstance *script, s32 isInitialCall) {
+    s32 var;
+    void* temp_s0 = load_asset_by_name("party_pokopi", &var);
+
+    decode_yay0(temp_s0, N(D_80247B00));
+    general_heap_free(temp_s0);
+
+    N(D_8024BA90).unk_00 = N(D_80247D00);
+    N(D_8024BA90).unk_08 = 0x96;
+    N(D_8024BA90).unk_0A = 0x69;
+    N(D_8024BA90).unk_0C = 2;
+    N(D_8024BA90).unk_04 = N(D_80247B00);
+    N(D_8024BA90).unk_10 = 1;
+    set_message_images(&N(D_8024BA90));
+    return ApiStatus_DONE2;
+}
+*/
```

Comment:
This doesn't match? Why not NON_MATCHING?

Reply:
I dunno, I didn't look at this file. I have a script that copies all the same functions to make it easier in the future. The rodata and symbols will be wrong, other than that it should match, that's the idea anyway.

Reply:
Ah, I see

Reply:
Ah, I see